### PR TITLE
CRYP-119: Fix add wallet loading spinner bug

### PR DESCRIPTION
**Pull request by narroarrow**
JIRA: https://cryptify.atlassian.net/browse/CRYP-119

image

### CODE REVIEW

File: packages/client/components/LoadingSpinner.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { View, Animated, Easing } from "react-native";
+
+const LoadingSpinner = () => {
```
Comment (jason-gerard) [inline packages/client/components/LoadingSpinner.tsx:4]:
```suggestion
export default function LoadingSpinner() {
```

File: packages/client/components/LoadingSpinner.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
+            <Animated.Image
+                style={{ transform: [{ rotate: spin }] }}
+                source={require("../assets/images/Spinner.png")}
+            />
+        </View>
+    );
+};
+
+export default LoadingSpinner;
```
Comment (jason-gerard) [inline packages/client/components/LoadingSpinner.tsx:31]:
```suggestion
```

File: packages/client/components/LoadingSpinner.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        outputRange: ["0deg", "360deg"],
+    });
+
+    return (
+        <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
+            <Animated.Image
+                style={{ transform: [{ rotate: spin }] }}
+                source={require("../assets/images/Spinner.png")}
+            />
+        </View>
```
Comment (jason-gerard) [inline packages/client/components/LoadingSpinner.tsx:27]: I wouldnt wrap the component in a view or any type of layout styling so you can leave it up to the client, the component that uses this, to decide how to position it.

I think for how we use it now the positioning should already be done, but double check and add any new layout stuff to where its used.

```suggestion
        <Animated.Image
            style={{ transform: [{ rotate: spin }] }}
            source={require("../assets/images/Spinner.png")}
        />
```

**Review by jason-gerard — APPROVED** [review]
Looks great

**Review by Andre-Ibrahim — APPROVED** [review]

**Review by Andre-Ibrahim — APPROVED** [review]

**Pull request merged**